packages/components/src/components/link/link.scss

```
@@ -2,14 +2,21 @@
 
 @use "@db-ui/foundations/build/scss/body-styles" as *;
 @use "@db-ui/foundations/build/scss/variables" as *;
+@use "@db-ui/foundations/build/scss/variables.global" as *;
 @use "@db-ui/foundations/build/scss/scaling-placeholder" as *;
 @use "@db-ui/foundations/build/scss/color-placeholder" as *;
 @use "@db-ui/foundations/build/scss/helpers/functions" as *;
 @use "@db-ui/foundations/build/scss/icon/icons.helpers" as *;
 @use "@db-ui/foundations/build/scss/helpers/component" as *;
 
+%db-link-height {
+	display: block;
```

```suggestion
	display: inline-block;
```

As the link is an inline element most likely, `inline-block` probably works better here.